package.json

```
@@ -1,12 +1,12 @@
 {
   "name": "requirejs-mock",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Deps update is not changing the contract - nothing should change, that is why its just a patch IMO

```suggestion
  "version": "1.0.2",
```